.circleci/config.yml

```
@@ -127,3 +127,12 @@ workflows:
           executor:
             name: hmpps/java
             tag: "19.0"
```

Should this be 21 as well ?

---

maybe even replace with this:

```suggestion
        - hmpps/gradle_owasp_dependency_check:
          slack_channel: probation_in_court_alerts_prod
          context:
            - hmpps-common-vars
          jdk_tag: "21.0"
          cache_key: "v3_0"
```

And you can delete the reference to `owasp: entur/owasp@0.0.10` orb

But then, you would have to start looking at the OWASP failures :-)
🐝